### DISCUSSION TITLE
colocate_with 属性的动态分区表报错

### DISCUSSION THREAD

**Discussion by kevinliukai** [apache/doris#11194] [A - General / Q&A]
1.0升级到1.1.0-rc05后，fe跟be节点状态都正常，历史colocate_with 属性的动态分区表报错：errCode = 2, detailMessage = Colocate tables distribution columns must have the same data type: sys_seq_id should be VARCHAR(*)，新创建的colocate_with 属性的表是正常的。

**Comment by dataalive** [comment]
schema have string type ?

**Reply by kevinliukai** [reply]
Schema not have string type，reset colocate_with is ok.
